modules/programs/git.nix

```diff
@@ -377,6 +403,19 @@ in {
         };
     })
 
+    (mkIf cfg.difftastic.enable {
+      home.packages = [ pkgs.difftastic ];
+
+      programs.git.iniContent =
+        let difftCommand = "${pkgs.difftastic}/bin/difft";
+        in {
+          core.pager = difftCommand;
+          interactive.diffFilter =
+            "${difftCommand} --color ${cfg.difftastic.color} --background ${cfg.difftastic.background}";
```

Comment:
This was heavily inspired by how the `Delta` option were defined and used, so I am not sure if this would work exactly as it should